avm/res/app/managed-environment/tests/e2e/max/main.test.bicep

```diff
@@ -66,6 +69,11 @@ module testDeployment '../../../main.bicep' = [
         }
       ]
       internal: true
+      dnsSuffix: 'contoso.com'
+      certificateKeyVaultProperties: {
+        identity: nestedDependencies.outputs.managedIdentityResourceId
+        keyVaultUrl: '${nestedDependencies.outputs.keyVaultUri}secrets/${split(nestedDependencies.outputs.certificateSecretUrl, '/')[4]}'
```

Comment:
By checking the deployment in Azure, can you confirm if the expected value is passed into the resource-group-level deployment called `.*-test-amemax-.*` nested in the subscription-level deployment `a-r-a-me-max-t1-.*` for the parameter `keyVaultUrl`

Reply:
I seem to have fixed the test run issue by fixing the naming issue.  However running the tests concurrently poses issues, running them one works

Reply:
False negative: tests run just fine. Must've been a hickup in my environment